java/piranha/src/main/java/com/uber/piranha/config/PiranhaMethodRecord.java

```diff
@@ -102,14 +87,7 @@ private static String getValueStringFromMap(Map<String, Object> map, String key)
    */
   @Nullable
   private static Integer getArgumentIndexFromMap(Map<String, Object> map) {
-    Object value = map.get(ARGUMENT_INDEX_KEY);
-    if (value instanceof Long) {
-      int argumentIndex = ((Long) value).intValue();
-      if (argumentIndex >= 0) {
-        return argumentIndex;
-      }
-    }
-    return null;
+    return PiranhaRecord.getArgumentIndexFromMap(map, ARGUMENT_INDEX_KEY);
```

Comment:
Again, feel free to just inline this, rather than making a new method just for passing `ARGUMENT_INDEX_KEY` :)